### DISCUSSION TITLE
Actions

### DISCUSSION THREAD

**Discussion by gavar** [codewriter-packages/UniMob#13] [Q&A] [answered]
Hi Everyone,

I very much like the project, I see it might bring great value to our team!

I have looked through the code but was not able to find a concept similar to [MobX Actions](https://mobx.js.org/actions.html).
I'm looking for something that can be used to mark the method for postponing updates so they are committed in batch to avoid triggering change on every property update done in the method. 

So what I'm looking for is exactly the first part

> 1. They are run inside [transactions](https://mobx.js.org/api.html#transaction). No reactions will be run until the outer-most action has finished, guaranteeing that intermediate or incomplete values produced during the action are not visible to the rest of the application until the action has been completed.

Would greatly appreciate best practices on that!

**Comment by vanifatovvlad** [comment, marked as answer]
Hi,

In MobX actions:
1.  They are postponing updates;
2.  They are untracked. 

In UniMob:
1. All reactions are always batched and executed on next frame ([similar discussion](https://github.com/codewriter-packages/UniMob/discussions/7#discussioncomment-1032579))
2. Any piece of code can be marked as untracked with [NoWatch](https://github.com/codewriter-packages/UniMob/blob/master/Runtime/Atom.NoWatch.cs)

so the Actions concept is not necessary.
